resource.images.iplayerwww/.gitignore

```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Please exclude

Reply:
Just to be sure: exclude the single line or the whole .gitignore?

Reply:
> Just to be sure: exclude the single line or the whole .gitignore?

The whole file.